src/m365/commands/login.ts

```diff
@@ -151,6 +197,88 @@ class LoginCommand extends Command {
     await this.initAction(args, logger);
     await this.commandAction(logger, args);
   }
+
+  private tokensForMultipleTenants(accessTokenValue: string | string[] | undefined, tenantValue: string | undefined): boolean {
+    const accessTokens = typeof accessTokenValue === "string" ? [accessTokenValue] : accessTokenValue as string[];;
+    let tenant = tenantValue || config.tenant;
+    let forMultipleTenants: boolean = false;
+
+    for (const token of accessTokens) {
+      const tenantIdInAccessToken = accessTokenUtil.accessToken.getTenantIdFromAccessToken(token);
+
+      if (tenant !== 'common' && tenant !== tenantIdInAccessToken) {
+        forMultipleTenants = true;
```

Comment:
We can simplify this implementation by changing this line to `return true` and then 214 to `return false`